src/jit/codegenlinear.h

```diff
@@ -131,6 +131,7 @@ void genFMAIntrinsic(GenTreeHWIntrinsic* node);
 void genLZCNTIntrinsic(GenTreeHWIntrinsic* node);
 void genPCLMULQDQIntrinsic(GenTreeHWIntrinsic* node);
 void genPOPCNTIntrinsic(GenTreeHWIntrinsic* node);
+void genX86SIMD_R_R_R(instruction ins, regNumber targetReg, regNumber op1Reg, regNumber op2Reg, var_types simdtype);
```

Comment:
Either this should have a name that's consistent with other `CodeGen` member names (e.g. `genTernarySIMD`) or move to instr.cpp (e.g. `inst_SIMD_R_R_R`), or perhaps it belongs in the emitter (e.g. `emitter::emit_SIMD_R_R_R`). I would suggest the latter. I would also omit the X86, and I would expect all of these to be defined under #ifdef _TARGET_XARCH_, as other targets will also eventually define `FEATURE_HW_INTRINSICS`

Reply:
> I would expect all of these to be defined under #ifdef TARGET_XARCH, as other targets will also eventually define FEATURE_HW_INTRINSICS

Good point. I will check other files (i.e. importer) to fix as well.